Rakefile

```diff
@@ -42,6 +42,8 @@ task setup_workspace: :environment do
     end
     File.write(target, content)
   end
+
+  puts "Workspace setup completed."
```

Comment:
I think this will still print when there is an error. I think we should only display this when no errors are thrown, and rephrase it as "Workspace setup completed successfully 🎉".

Reply:
@julianguyen Agree 👍 
I have improved the exception handling in this task. Could you please review it again?